concent_api/middleman/utils.py

```diff
@@ -0,0 +1,78 @@
+from typing import Dict
+
+import asyncio
+
+from logging import getLogger
+from logging import Logger
+
+from middleman.constants import ResponseQueueItem
+
+STANDARD_ERROR_MESSAGE = "This connection has been already added"
+WRONG_TYPE_ERROR_MESSAGE = "QueuePool can only contain mapping between connection ID (int) and response queue (asyncio.Queue)"
+
+
+class QueuePool(dict):
```

Comment:
Shouldn't it define `__getitem__()` too ?

Reply:
not really; there is no need for any additional behavior during getting an item